src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.iOS.cs

```diff
@@ -62,6 +62,11 @@ private void InitializeNativePanel(NativeListViewBase panel)
 			_callbackSubscriptions.Disposable = disposables;
 		}
 
+		partial void OnDataContextChangedPartial()
+		{
+			NativePanel?.UpdateHeaderAndFooter();
+		}
```

Comment:
- What about other platforms?
- Is it possible to have a test for this?

Reply:
Should the issue be kept open to track the other platforms?

Reply:
@Youssef1313, this bug doesn't exist on Android because a similar code is already in place. It's only iOS. I didn't test other platforms.

Reply:
I'm not sure how to run the tests locally, can you help with that please?

Reply:
You can run the tests as [documented here](https://platform.uno/docs/articles/uno-development/creating-runtime-tests.html#running-tests-locally). It's basically an entry in the samples app that is a unit tests runner.